packages/manager/apps/web/client/app/domain/webhosting/order/steps/dnsConfiguration/domain-webhosting-order-steps-dnsConfiguration.component.js

```diff
@@ -0,0 +1,11 @@
+import controller from './domain-webhosting-order-steps-dnsConfiguration.controller';
+import template from './domain-webhosting-order-steps-dnsConfiguration.html';
+
+export default {
+  controller,
+  name: 'ovhManagerWebDomainWebhostingOrderStepsDnsConfiguration',
+  require: {
+    stepper: '^ovhManagerWebDomainWebhostingOrderSteps',
```

Comment:
I'm not too fond of components requiring parent controller ... This prevents reusability

Reply:
These are subcomponents of the whole stepper one, they are not intended to be reuse elsewhere, without their parent.

I've made all these children components in order to split the template/controller of the parent.